mne/inverse_sparse/mxne_optim.py

```diff
@@ -192,6 +192,9 @@ def dgap_l21(M, G, X, active_set, alpha, n_orient):
         The regularization parameter.
     n_orient : int
         Number of dipoles per locations (typically 1 or 3).
+    primal_only : bool
+        Only computes and returns the primal value. Useful when only the
+        primal is needed.
 
     Returns
     -------
```

Comment:
I guess you need to update the Returns section. I am not a huge fan that a function called dgap_... does not return a dual gap...

Reply:
Same here. I didn't want to duplicate code, but I can create a function primal_l21 if it is more appropriate.